packages/@aws-cdk/aws-apprunner/test/docker.assets/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM nginx:latest
```

Comment:
Nit: can we fully qualify the image name here? Bonus points if that base image is available from ECR public... :)

Reply:
OK. Will change to https://gallery.ecr.aws/aws-containers/hello-app-runner.